dsn/testdata/delayed_dsn.raw

```diff
@@ -0,0 +1,27 @@
+MIME-Version: 1.0
```

Comment:
Given we are introducing a number of test data files, I think making a `testdata/dsn` subdirectory would be a good idea.

Reply:
do you want them in `enmine/testdata/dsn/` or in `enmime/dsn/testdata/dsn/`?

Reply:
Nevermind on this one, I didn't realize these were already in the `dsn` pkg directory, should be fine as is.